### DISCUSSION TITLE
App can't be accessed outside of the Saleor Dashboard ❌ The view can only be displayed in the iframe.

### DISCUSSION THREAD

**Discussion by manhtd98** [saleor/saleor#10407] [General]
I try to run saleor-app-template in 3001 port. And got this error when try to view app on `http://localhost:3001/`

**Comment by karolkielecki** [comment]
Hi @manhtd98, could you please provide more details? A step-by-step process will help us to reproduce the issue.

**Comment by manhtd98** [comment]
on repo `https://github.com/saleor/saleor-app-template` i just run :
```
pnpm install
pnpm run dev
```
And go to localhost:3001
i got this messenge `App can't be accessed outside of the Saleor Dashboard x The view can only be displayed in the iframe`

**Comment by krzysztofwolski** [comment]
To access the app you need to install it to your Saleor instance.

Instructions for using the CLI:
https://learn.saleor.io/cli/creating-apps/

If you are running Saleor locally, here are the docs:
https://docs.saleor.io/docs/3.x/developer/extending/apps/installing-apps

**Comment by Cloud11PL** [comment]
I see you're not using CLI, take a look at this link: https://github.com/saleor/saleor-app-template#local-development-without-cli
